src/api/lambda.ts

```diff
@@ -40,7 +40,7 @@ export const lambdaCall = (fnName: string, payload: any): Promise<any> => new Pr
 
       lambda.invoke(params, (err, data) => {
         if (err) {
-          logger.error('err', err);
+          logger.error(err);
```

Comment:
Should we add a prefix? Or does it automatically add information about where the error was logged from?

Reply:
Yes we should, it does not add information automatically.

Done in 73bd6bf3c8dbab8b7a5598b6ea1cb8abd1c558c1